packages/storefront-lwc/src/modules/commerce/productSearchResults/productSearchResults.html

```diff
@@ -9,45 +9,63 @@
+                                        <div class='reset btn btn-block btn-outline-primary' onclick={resetRefinements}>
+                                            Reset
+                                        </div>
                                     </div>
+
+                                    <aside>
+                                        <div class='refinements'>
+                                            <template for:each={refinementgroups} for:item="refinementgroup">
+                                                <commerce-refinement-group ontogglerefinement={toggleRefinementHandler}
+                                                    key={refinementgroup.label} refinementgroup={refinementgroup}>
+                                                </commerce-refinement-group>
+                                            </template>
+                                        </div>
+                                    </aside>
                                 </div>
 
-                                <aside>
-                                    <div class='refinements'>
-                                        <template for:each={refinementgroups} for:item="refinementgroup">
-                                            <commerce-refinement-group ontogglerefinement={toggleRefinementHandler}
-                                                                       key={refinementgroup.label}
-                                                                       refinementgroup={refinementgroup}>
-                                            </commerce-refinement-group>
-                                        </template>
+
+                                <div class='col col-lg-9 product-list-grid'>
+                                    <div class='container'>
+                                        <commerce-product-list ontogglerefinementbar={toggleRefinementBarHandler}
+                                            onupdatesort={updateSortHandler} products={products} sort={sortingOptions}>
+                                        </commerce-product-list>
                                     </div>
-                                </aside>
+                                </div>
                             </div>
+                        </template>
+                    </template>
 
-                            <div class='col col-lg-9 product-list-grid'>
-                                <div class='container'>
-                                    <commerce-product-list ontogglerefinementbar={toggleRefinementBarHandler}
-                                                          onupdatesort={updateSortHandler}
-                                                          products={products}
-                                                          sort={sortingOptions}>
-                                    </commerce-product-list>
+                    <template if:false={flag}>
```

Comment:
This template shouldn't show based purely on this flag, essentially it should be `flag && !loading`. This can be done LWC by using a getter. The easiest way to do it would probably be:

```js
get hasResults() {
  return this.products.length && !this.loading && !this.error;
}
```

Then within your template you can show the loading, error, and normal state.